code/modules/antagonists/clockcult/clock_items/clockwork_armor.dm

```diff
@@ -70,6 +70,7 @@
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 	armor = list("melee" = 60, "bullet" = 70, "laser" = -25, "energy" = 0, "bomb" = 60, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 100)
 	allowed = list(/obj/item/clockwork, /obj/item/clothing/glasses/wraith_spectacles, /obj/item/clothing/glasses/judicial_visor, /obj/item/mmi/posibrain/soul_vessel, /obj/item/reagent_containers/food/drinks/bottle/holyoil)
+	tauric = TRUE		//Citadel Add for tauric hardsuits
```

Comment:
change to `mutantrace_variation = STYLE_DIGITIGRADE|STYLE_SNEK_TAURIC`
the tauric var is no longer used